### DISCUSSION TITLE
How to change order of the Children in Multi-Select custom ValueContainer?

### DISCUSSION THREAD

**Discussion by Wdifulvio523** [JedWatson/react-select#5013] [Q&A]
custom value container renders children, which is split into the selected values and the Input (cursor to type).
![Screen Shot 2022-01-18 at 6 34 48 PM](https://user-images.githubusercontent.com/38021468/150035801-159c84e8-330f-41c7-9212-c2260790cb28.png)

 I want to be able to reverse this so that the input shows up before the selected values. How can this be done?

**Comment by Rall3n** [comment]
Hello @Wdifulvio523,

you can easily achieve this layout by using the styling framework of the `Select` component. We mostly use [Flexbox](https://developer.mozilla.org/en-US/docs/Learn/CSS/CSS_layout/Flexbox) for the layouts, so you can easily change the order of values and input element by using the `order` property.

```jsx
<Select
  styles={{
    input: (provided) => ({
      ...provided,
      order: -1,
      flex: 0 // To prevent it from taking up all the space by default
    })
  }}
/>
```
